File: app/src/main/java/com/utynote/components/search/data/geojson/entities/Feature.java

package com.utynote.components.search.data.geojson.entities;

import java.util.List;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Feature {

    @SerializedName("type")
    @Expose
    public String type;
    @SerializedName("geometry")
    @Expose
    public Geometry geometry;
    @SerializedName("properties")
    @Expose
    public Properties properties;
    @SerializedName("bbox")
    @Expose
    public List<Double> bbox = null;

}

File: app/src/main/java/com/utynote/components/search/data/geojson/entities/Feature.kt
package com.utynote.components.search.data.geojson.entities

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Feature {

    @SerializedName("type")
    @Expose
    var type: String? = null
    @SerializedName("geometry")
    @Expose
    var geometry: Geometry? = null
    @SerializedName("properties")
    @Expose
    var properties: Properties? = null
    @SerializedName("bbox")
    @Expose
    var bbox: List<Double>? = null

}
